src/pages/api/auth/logout-callback.ts

```diff
@@ -20,7 +20,7 @@ async function callback(req: NextApiRequest, res: NextApiResponse) {
   if (csrf !== req.cookies['auth-csrf']) return fail('CSRF validation failed')
 
   try {
-    const token = JSON.parse(req.cookies['auth-token']) as Token
```

Comment:
Although I haven't changed anything in this file, it started to complain about 
```console
src/pages/api/auth/logout-callback.ts:23:30 - error TS2345: Argument of type 'string | undefined' is not assignable to parameter of type 'string'.
  Type 'undefined' is not assignable to type 'string'.

23     const token = JSON.parse(req.cookies['auth-token']) as Token
                                ~~~~~~~~~~~~~~~~~~~~~~~~~
```
So I added the !, but I'm not sure what caused it and if we could solve that without changing this file for now.

Reply:
Thanks for your comment. That explain some things 😄.
I had to solve conficts in `yarn.lock`, so I deleted it and ran `yarn install`. 
But I try it other way the next time...

Reply:
I allowed myself to revert the lock changes so the [files tab](https://github.com/serlo/frontend/pull/1779/files) loads faster again :)

Reply:
@elbotho Maybe we want to revert this line to 
`const token = JSON.parse(req.cookies['auth-token'])...` without the !, so that we somehow don't change the behavior related to hydra for now. I can do that later